### DISCUSSION TITLE
Schneider PowerTag spamming "Unknown attribute ... in cluster ..."

### DISCUSSION THREAD

**Discussion by Thomas-Wimmer** [Koenkk/zigbee2mqtt#22547] [Devices]
Hello,

My [Schneider PowerTag A9MEM1570](https://www.zigbee2mqtt.io/devices/A9MEM1570.html) spams the log (and on the main screen) of Z2M with the following messages:

`Unknown attribute 19200 in cluster haElectricalMeasurement`
`Unknown attribute 19456 in cluster haElectricalMeasurement`
`Unknown attribute 19712 in cluster haElectricalMeasurement`
(and maybee some other numbers)
`Unknown attribute 16911 in cluster seMetering`
`Unknown attribute 16403 in cluster seMetering`
`Unknown attribute 16655 in cluster seMetering`
(also some other numbers)

![grafik](https://github.com/Koenkk/zigbee2mqtt/assets/122106679/e5937c52-d69e-4832-9ea6-ed45df99ba4b)

I joined the PowerTag via an [IKEA TRADFRI signal repeater](https://www.zigbee2mqtt.io/devices/E1746.html), with an SLZB-06 (with the CC2652P chip) as the coordinator.

How can I fix (or mute if unfixable) this?

Thanks,

Thomas

**Comment by Koenkk** [comment]
Will be fixed in the next z2m release

**Reply by Thomas-Wimmer** [reply]
Thanks for the quick reply
